recipes/libyuv/all/patches/001-cmake.patch

```diff
@@ -0,0 +1,55 @@
+--- CMakeLists.txt
++++ CMakeLists.txt
+@@ -6,6 +6,9 @@
+ CMAKE_MINIMUM_REQUIRED( VERSION 2.8 )
+ OPTION( TEST "Built unit tests" OFF )
+ 
++include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
```

Comment:
Export it to a CMakeLists.txt, it's a common pattern here in CCI: https://github.com/conan-io/conan-center-index/blob/ccd4ba40a2f39d03108c3bb25bae80c1410e8114/recipes/kcp/all/CMakeLists.txt

Reply:
fixed